script.js

```diff
@@ -1,200 +1,70 @@
-// output is hidden by default
-document.getElementById('output').style.visibility = 'hidden';
+const inputEl = document.querySelector('#weightInput');
+const outputEl = document.querySelector('.output');
+const unitSelectorEl = document.querySelector('.unit-selector');
+const placeholderEl = inputEl.placeholder;
+const outputValues = document.querySelectorAll('.card-output');
+const weightTypeEl = document.querySelector('#weightType');
+const gramsOutputEl = document.querySelector('#gramsOutput');
+const lbsOutputEl = document.querySelector('#lbsOutput');
+const ozOutputEl = document.querySelector('#ozOutput');
+const kgOutputEl = document.querySelector('#kgOutput');
+const kilogramsEl = document.querySelector('#kilograms');
+const gramsEl = document.querySelector('#grams');
+const poundsEl = document.querySelector('#lbs');
+const ouncesEl = document.querySelector('#oz');
 
-//DECLARE VARIABLES
-// body
-const body = document.querySelector("body");
-//weight input
-const input = document.querySelector("#weightInput");
-// conversion output
-const output = document.querySelector("#output");
-// placeholder 
-const placeholder = input.placeholder;
-// select field
-const weightType = document.querySelector("#weightType")
-// output - grams 
-const gramsOutput = document.querySelector("#gramsOutput");
-// output - pounds
-const lbsOutput = document.querySelector("#lbsOutput");
-// output - ounces
-const ozOutput = document.querySelector("#ozOutput");
-// output - kilograms
-const kgOutput = document.querySelector("#kgOutput");
-// select field - kilograms
-const kilograms = document.querySelector("#kilograms");
-// select field - grams
-const grams = document.querySelector("#grams");
-// select field - lbs
-const pounds = document.querySelector("#lbs")
-// select field - oz
-const ounces = document.querySelector("#oz")
+function hideConversions() {
+    outputEl.style.visibility = 'hidden';

```

Comment:
Can you achieve the same effect by adding a class like `hidden` on the element instead of changing the style attribute?